lib/kitchen/verifier/pester.rb

```diff
@@ -211,7 +221,11 @@ def install_command_script
                   Install-Module Pester -Force
               }
               else {
-                  if (-not (Test-Module -Name PsGet)){
+                  if (-not (Test-Module -Name PsGet)){ # We should get rid of this. 
+                      # installing PSGet from someone we don't know's github raw is scary bad.
+                      # We don't want PSGet but maybe PowerShellget instead.
+                      # Maybe install from nupkg directly from a nuget feed (Odata), doing the nupkg unzip ourselves?
+                      # and we need PowerShellGet, v2 or v3
```

Comment:
Yep, agreed. I left PsGet in for the time being because I wasn't sure if PowerShellGet could be used downlevel, but as far as I'm aware it's usable down to v2 IIRC, provided we can bootstrap it.

Reply:
And I guess you need it to support PSv2? 🙄

Reply:
I wish I could say no, I don't like dealing with PSv2, but... yep. 😔

Reply:
as discussed, tested with PS4 only in 2012r2...